setup.cfg

```diff
@@ -1,10 +1,10 @@
 [aliases]
 
 # All tests in parallel, minus any requiring docker
-test             = pytest --addopts "-n auto -m 'not dockertest'"
+test             = pytest --addopts "-n auto --ignore benchmarks -m 'not dockertest'"
 
 # Test _everything_
-testall          = pytest --addopts "--ignore benchmarks"
+testall          = pytest --addopts "-n auto --ignore benchmarks -m 'not dockertest'"
```

Comment:
`testall`  seems suspiciously similar to `test` now, and neither of them tests all...

Reply:
WIP